ui/hidden_rewind_button.js

```diff
@@ -0,0 +1,117 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.ui.HiddenRewindButton');
+
+goog.require('shaka.ui.Element');
+goog.require('shaka.util.Timer');
+goog.require('shaka.ui.Enums');
+goog.require('shaka.util.Dom');
+
+goog.requireType('shaka.ui.Controls');
+
+/**
+ * @extends {shaka.ui.Element}
+ * @final
+ * @export
+ */
+shaka.ui.HiddenRewindButton = class extends shaka.ui.Element {
```

Comment:
Scanning over these two classes, they seem to be very similar.
Could you make a superclass that both of these inherit from?

Reply:
Done!